src/neuroglancer/mesh/backend.ts

```diff
@@ -443,18 +459,20 @@ export class MeshLayer extends SegmentationLayerSharedObjectCounterpart implemen
     const basePriority = getBasePriority(visibility);
     const {source, chunkManager} = this;
     forEachVisibleSegment3D(this, objectId => {
-      let manifestChunk = source.getChunk(objectId);
+      // if objectId exists in newRootSegments, do not verify if mesh fragments exist
+      let manifestChunk = source.getChunk(objectId, !this.newRootSegments!.has(objectId));
```

Comment:
This one is also very specific to graphene, but will be used by all data formats with meshes...  Do you already have an idea how the verifyFragment function might look? Or rather, the do-not-verify function for the user-generated roots that are re-using fragments?

Reply:
its in `datasource/graphene/backend.ts`, the main issue is manifest call and mesh fragment rendering are in different files